### DISCUSSION TITLE
How to save dynamic input values

### DISCUSSION THREAD

**Discussion by AsemAbbas** [livewire/livewire#4207] [1. Help] [answered]
in my form i make a dynamic input but i can't save the value as array

this is my view :

![image](https://user-images.githubusercontent.com/80175112/141510143-8da3bc41-3184-4a23-9ca7-48e3bbcb0b91.png)

and this is my component:

![image](https://user-images.githubusercontent.com/80175112/141510291-e7a5b673-f1fa-436a-8a38-d7bf5c1111bb.png)

test is not array:

![image](https://user-images.githubusercontent.com/80175112/141509902-5671d636-28d7-4d64-b47b-657d32000fc0.png)

**Comment by zepfietje** [comment, marked as answer]
`wire:model.defer="test.{{ $loop->index }}"` should work.

Let me know if it fixes your issue!

**Reply by AsemAbbas** [reply]
omg! thank you so much it`s work
